eql/__init__.py

```diff
@@ -66,7 +66,7 @@
     Walker,
 )
 
-__version__ = '0.9.19'
+__version__ = '0.9.20'
```

Comment:
Removing backwards compatibility with Python 2, should we bump major or minor instead?